.circleci/config.yml

```diff
@@ -69,10 +52,14 @@ jobs:
           name: Setup e2e Environment
           command: |
             cp /binaries/kind /usr/local/bin/kind
+            curl -LO https://github.com/kubernetes-sigs/kind/releases/download/0.2.1/kind-darwin-amd64
+            chmod +x kind-darwin-amd64
+            mv kind-darwin-amd64 /usr/local/bin
```

Comment:
Is this running in a linux container with a Mac built binary?

Reply:
It does look that way.  I'm not sure how this is working right now.  🤔

Reply:
Also this will make a file called `/usr/local/bin/kind-darwin-amd64` but the script below references just `kind` as the binary name. I think maybe your restore cache may have been leading to a false positive in the testing?

Reply:
I think you're right.  I have fixed both of those in my latest iteration.